system_test/aest_sync_SUITE.erl

```diff
@@ -61,12 +63,48 @@
     source  => {pull, "aeternity/epoch:local"}
 }).
 
+
+-define(NET1_NODE1, #{
```

Comment:
Is it such that if you mention [net1] in this setup, then it won't be connected to other networks, but you may still have nodes to that network? A comment would be in place.
This is different semantics from when you mention a node that is not defined, because then you get an error message.